### PR TITLE
[PLT-5158] Added inverted button type

### PR DESCRIPTION
**Pull request by bolinkd**
## Description
Added inverted button type for Search Page Header

## Screenshots
Screen Shot 2020-04-28 at 15 57 29

## Where to Start
packages/riipen-ui/src/components/Button.jsx

### CODE REVIEW

File: packages/riipen-ui/src/components/Button.jsx [hunk 1]
```diff
@@ -400,6 +401,88 @@ class Button extends React.Component {
...
+          .inverted-tertiary {
+            color: ${theme.palette.tertiary.main};
+          }
+          .inverted-tertiary:hover::before {
+            background-color: ${theme.palette.tertiary.main};
+          }
+          .inverted-tertiary::after {
+            border-color: ${theme.palette.tertiary.main};
+          }
+          .inverted-white {
```
Comment (ericauv) [inline packages/riipen-ui/src/components/Button.jsx:458]:
```suggestion
          .inverted-black {
```
To maintain consistency of naming, I think we should continue using the `color` as `${color}` in `inverted-${color}`

File: packages/riipen-ui/src/components/Button.jsx [hunk 2]
```diff
@@ -400,6 +401,88 @@ class Button extends React.Component {
...
+            color: ${theme.palette.common.black};
+          }
+          .inverted-white:hover::before {
+            background-color: ${theme.palette.grey[100]};
+          }
+          .inverted-white::after {
+            border-color: ${theme.palette.common.black};
+          }
```
Comment (ericauv) [inline packages/riipen-ui/src/components/Button.jsx:466]:
```suggestion
          .inverted-black {
            color: ${theme.palette.common.black};
          }
          .inverted-black:hover::before {
            background-color: ${theme.palette.grey[100]};
          }
          .inverted-black::after {
            border-color: ${theme.palette.common.black};
          }
```

Comment (ericauv) [reply packages/riipen-ui/src/components/Button.jsx:466]: Would need to add `black` to `propTypes:{color}` also

Comment (ericauv) [reply packages/riipen-ui/src/components/Button.jsx:466]: / For `inverted-white`, I think it should have `{backgroundColor: black, color: white}`

Comment (bolinkd) [reply packages/riipen-ui/src/components/Button.jsx]: I dont think the use case for this is very large, I am going to remove it for now and let someone build it when the use case presents itself

**Pull request merged**